server/src/main/java/org/apache/uniffle/server/ShuffleServerConf.java

```diff
@@ -89,6 +89,13 @@ public class ShuffleServerConf extends RssBaseConf {
           .defaultValue(10 * 1000L)
           .withDescription("Direct memory usage tracker interval to MetricSystem (ms)");
 
+  public static final ConfigOption<Long> SERVER_NETTY_PENDING_TASKS_NUM_TRACKER_INTERVAL =
+      ConfigOptions.key("rss.server.netty.pending.tasks.num.updateMetricsIntervalMs")
```

Comment:
Maybe `NETTY_PENDING_TASKS_NUM_TRACKER_INTERVAL`?

Reply:
I followed the name `rss.server.netty.directMemoryTracker.memoryUsage.updateMetricsIntervalMs`.

Maybe `rss.server.netty.pending.tasks.num.polling.interval`? If we continue to simplify the naming of this variable, I'm afraid it might become difficult to understand.

Reply:
How about `rss.server.netty.metrics.pendingTaskNumPollingInterval`?

Reply:
OK. Done.